.github/workflows/issue.yml

```
@@ -24,12 +24,30 @@ jobs:
         summary: « [hugr] ${{ github.event.issue.title }}»
         description: ${{ github.event.issue.html_url }}
         fields: '{"labels": ["hugr"]}'
+    - name: set_priority_var
+      env:
+        GH_PRIORITY: ${{ github.event.issue.label }}
+      run: |
+        MY_RESULT=""
+        case ${GH_PRIORITY} in
+          P-low)
+            MY_RESULT=echo Low;;
+          P-medium)
+            MY_RESULT=Medium;;
+          P-high)
+            MY_RESULT=High;;
+          P-critical)
+            MY_RESULT=Highest;;
+        esac
+        echo $MY_RESULT >> $GITHUB_OUTPUT
+
     - name: Create Task
-      uses: atlassian/gajira-create@v3.0.1
+      env:
+        JIRA_PRIORITY: ${{ steps.set_priority_var.outputs.JIRA_PRIORITY }}
       if: "! contains(github.event.issue.labels.*.name, 'bug')"
```

This works as-is, but the linter works better with `{{`s.
```suggestion
      if: ${{! contains(github.event.issue.labels.*.name, 'bug')}}
```
(you may want to change it on the other `if contains` too)